tests/testthat/test-renderer4-geoms-zoom.R

```diff
@@ -0,0 +1,97 @@
+acontext("zooming of geoms")
+
+# Plots with axis updates
+mtcars$cyl <- as.factor(mtcars$cyl)
+
+no_updates <- ggplot()+geom_point(aes(mpg, disp, 
+                                      colour=cyl), 
+                                  data = mtcars)
+update_x <- no_updates+
+  theme_animint(update_axes=c("x"))
+update_y <- no_updates+
+  theme_animint(update_axes=c("y"))
+update_xy <- no_updates+
+  theme_animint(update_axes=c("x","y"))
+
+viz <- (list(g=no_updates, 
+             g1=update_x, 
+             g2=update_y, 
+             g3=update_xy))
+# We only update axes for single selectors
+viz$selector.types = list(cyl="single")
+
+info <- animint2HTML(viz)
+
+# Update selection and get HTML
+clickID(c("plot_g_cyl_variable_8"))
+Sys.sleep(0.5)
+info$html_updated1 <- getHTML()
+
+# Update selection and get HTML
+clickID(c("plot_g_cyl_variable_4"))
+Sys.sleep(0.5)
+info$html_updated2 <- getHTML()
+
+## Get ranges of geoms
+no_updates_nodes1 <-
+  getNodeSet(info$html_updated1, '//g[@class="geom1_point_g"]//circle')
+no_updates_nodes2 <-
+  getNodeSet(info$html_updated2, '//g[@class="geom1_point_g"]//circle')
+no_updates_attrs1 <- sapply(no_updates_nodes1, xmlAttrs)[c("cx", "cy"), ]
+no_updates_attrs2 <- sapply(no_updates_nodes2, xmlAttrs)[c("cx", "cy"), ]
+no_updates_xranges1 <- range(as.numeric(no_updates_attrs1[1, ]), na.rm = T)
+no_updates_xranges2 <- range(as.numeric(no_updates_attrs2[1, ]), na.rm = T)
+no_updates_yranges1 <- range(as.numeric(no_updates_attrs1[2, ]), na.rm = T)
+no_updates_yranges2 <- range(as.numeric(no_updates_attrs2[2, ]), na.rm = T)
```

Comment:
hey @faizan-khan-iit can you please clarify this test by removing repetition? for example you could make a helper function `get_pixel_ranges` that takes the geom class name as argument...

```
get_pixel_ranges("geom2_point_g1")
get_pixel_ranges("geom3_point_g2")
```